astropy/wcs/tests/test_wcsprm.py

```diff
@@ -51,26 +51,26 @@ def test_cd():
     w = _wcs.Wcsprm()
     w.cd = [[1, 0], [0, 1]]
     assert w.cd.dtype == np.float
-    assert w.has_cd() == True
+    assert w.has_cd()
     assert_array_equal(w.cd, [[1, 0], [0, 1]])
     del w.cd
-    assert w.has_cd() == False
+    assert w.has_cd() is False
```

Comment:
I think this was an oversight from my side and those should have been ``is True`` as well to test their booleanness.